source/security/enterprise-authentication.txt

```diff
@@ -0,0 +1,126 @@
+.. _cpp-enterprise-auth:
+
+Enterprise Authentication Mechanisms
+====================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: ldap, tls, security
+
+Overview
+--------
+
+MongoDB Enterprise Edition includes authentication mechanisms that aren't
+available in MongoDB Community Edition. 
+In this guide, you can learn how to authenticate to MongoDB by using these
+authentication mechanisms. To learn about the other authentication mechanisms available
+in MongoDB, see :ref:`<cpp-auth>`.
+
+.. _cpp-kerberos:
+
+Kerberos
+--------
+
+The Generic Security Services API (GSSAPI) provides an interface for Kerberos
+authentication.
+
+.. note::
+
+   To authenticate with GSSAPI, you must build the driver with SASL support.
```

Comment:
```suggestion
   To authenticate with GSSAPI, you must build the C driver with SASL support.
```

Suggest clarifying the `ENABLE_SASL` option is intended for the C driver (not C++ driver). The C++ driver wraps the C driver. Auth is performed within the C driver.